metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -144,21 +148,41 @@ def extract_dbt_entities(
             if catalog_node is None:
                 report.report_warning(
                     key,
-                    f"Entity {dbtNode.dbt_name} is in manifest but missing from catalog",
+                    f"Entity {key} is in manifest but missing from catalog",
                 )
 
             else:
 
-                dbtNode.materialization = all_catalog_entities[key]["metadata"][
+                materialization = all_catalog_entities[key]["metadata"][
```

Comment:
Why do we reset materialization here? Shouldn't ```node["config"]["materialized"]``` have it already? Are there certain cases when it won't be populated, and in those cases the node will be equal to None?

Reply:
It looks like this is a fallback for instances where materialization is not specified for the node (since `config` can sometimes be missing) but is present in the catalog.

Reply:
So can't this overwrite the value of materialization to None?